### DISCUSSION TITLE
[V3][UX] - Improve tabindex on login page

### DISCUSSION THREAD

**Discussion by billmn** [filamentphp/filament#7264] [General]
Currently on the login page (with registration and password reset active), the focus sequence with TAB is as follows:

1. Email field
2. Forgot password link
3. Password field
4. Remember checkbox
5. Submit button
6. Registration link (because the autofocus is on the email field)

What about moving the focus of the forgot password link after the submit button?
The sequence would then become:

1. Email field
2. Password field
3. Remember checkbox
4. Submit button
5. _**Forgot password link**_  <---
6. Registration link (because the autofocus is on the email field)

An example is the [login page of Github](https://github.com/login).

**Comment by zepfietje** [comment]
Agree. I've seen this issue myself, but haven't found the time to fix it.
Would you be able to create a pull request that fixes it?
I'm happy to review and merge it then.

Also, I think this is more of a bug than a discussion.
So next time you can just create a bug report for it.
Increases the chances of me seeing it as I don't personally monitor discussions on GitHub. 🙂

**Reply by billmn** [reply]
Absolutely!

I've looked into it, and handling tab order is not particularly easy (especially with cross browser support and avoiding the use of tabindex).

However, I am sending you a PR where the focus on "hint" is subsequent to the focus of the related field.

I did not open an issue because I was not convinced it was really a problem but more of an improvement. In any case, for next time, I will consider opening an issue :)
